Sources/Turf/Feature.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+#if !os(Linux)
+import CoreLocation
+#endif
+
+
+public struct Feature: GeoJSONObject {
+    public var type: FeatureType = .feature
+    public var identifier: FeatureIdentifier?
+    public var properties: [String : AnyJSONType]?
```

Comment:
I know this pattern was introduced before this PR, but I feel like `[String: AnyJSONType]` is a code-smell that we should remove ASAP. The `AnyJSONType` class doesn't really buy you anything -- You'd still have to cast `someAnyJSONTypeObject.jsonValue as? String` to get the correct type-safety.

If that's the case, we may as well represent JSON arrays as `[String: Any]`, and let developers conditionally-cast for the type they're looking for, as needed. `let someDouble = someJSONValue as? Double` is a very familiar pattern in swift API's.

However, this may be out-of-scope for this PR, and may be better served in a fast follow. @1ec5 @frederoni what say you?

Reply:
I think `AnyJSONType` was first introduced to facilitate decoding, but we only support JSONDecoder anyways, so we can be sure that decoding a `[String: Any?]` dictionary would contain only JSON-representable values. (Note that it should be `[String: Any?]`, not `[String: Any]`, because `null` is a valid JSON type.)

@JThramer, can you open a separate issue tracking that issue? I think @Udumft can make progress without having to consider this issue for now.

Reply:
Groovy, I'll go ahead and approve and cut the FF ticket.

Reply:
Ticket opened: https://github.com/mapbox/turf-swift/issues/94